render/detailed_node.go

```diff
@@ -74,6 +73,22 @@ func (t tables) Less(i, j int) bool { return t[i].Rank > t[j].Rank }
 // MakeDetailedNode transforms a renderable node to a detailed node. It uses
 // aggregate metadata, plus the set of origin node IDs, to produce tables.
 func MakeDetailedNode(r report.Report, n RenderableNode) DetailedNode {
+	tables := tables{}
```

Comment:
All the connections-table-related code is getting big enough to justify its own function now, don't you think?